mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -122,14 +122,14 @@ mlir::rock::requiredPadding(Attribute params, GemmSize gemmSize,
   int64_t kPerBlock, mPerBlock, nPerBlock;
   int64_t kPack = 1;
   if (auto generalParams = params.dyn_cast<GeneralGemmParamsAttr>()) {
-    kPerBlock = math_util::lcm(generalParams.getKPerBlock(), extraPadding->k);
-    mPerBlock = math_util::lcm(generalParams.getMPerBlock(), extraPadding->m);
-    nPerBlock = math_util::lcm(generalParams.getNPerBlock(), extraPadding->n);
+    kPerBlock = generalParams.getKPerBlock() * extraPadding->k;
```

Comment:
Why this change?

Reply:
The change was related to https://github.com/ROCm/rocMLIR-internal/issues/1411.

Now, I came up to a better way to do padding for the Split-K, which require neither `math_util::lcm` nor `generalParams.getKPerBlock() * extraPadding->k`. I reverted this changes to the state as it was before `Split-K`